rdt-open-reader/src/main/java/io/ona/rdt/domain/Patient.java

```diff
@@ -10,23 +10,28 @@
 
     private String patientName;
     private String patientSex;
+    private String address;
     private String baseEntityId;
     private String patientId;
 
-    public Patient(String patientName, String patientSex, String baseEntityId) {
+    public Patient(String patientName, String patientSex, String baseEntityId, String address) {
         this.patientName = patientName;
         this.patientSex = patientSex;
+        this.address = address;
         this.baseEntityId = baseEntityId;
     }
 
-    public Patient(String patientName, String patientSex, String baseEntityId, String patientId) {
-        this(patientName, patientSex, baseEntityId);
+    public Patient(String patientName, String patientSex, String baseEntityId, String patientId,  String address) {
```

Comment:
For backward compatibility, add a new constructor instead of modifying the existing one.